src/parameters/all_parameters.cpp

```diff
@@ -284,6 +284,10 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       dealii::Patterns::Bool(),
                       "Outputs the surface solution vtk files. False by default");
 
+    prm.declare_entry("do_renumber_dofs", "true",
+                      dealii::Patterns::Bool(),
+                      "Flag for renumbering DOFs. True by default. Set to false if doing 3D unsteady flow simulations.");
```

Comment:
```suggestion
                      "Flag for renumbering DOFs using Cuthill-McKee renumbering. True by default. Set to false if doing 3D unsteady flow simulations.");
```

Reply:
Maybe add a description to the effect of "... renumbering increases memory use..." so that a new student could understand easier when to set this.

Also, is this something which could be set according to already-existing parameters? I don't completely understand in what case we want renumbering. But if it's associated with AD runs, maybe we could use `ode_solver_type` as an indicator? I'm suggesting this because the more parameters we have, the more likely we are to make mistakes while setting them. (I know I've forgotten to change parameters many times, causing a useless result!)